File: 1. Two Sum/solution.swift
class Solution {
  func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
    guard nums.count > 1 else { return [] }
    var seen: [Int : Int] = [:]
    for i in 0..<nums.count {
      let val = nums[i]
      let numToFind = target - val
      if let idx = seen[numToFind] {
        return [idx, i]
      }
      seen[val] = i
    }
    return []
  }
}

File: 1. Two Sum/solution.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        dict = {}
        for i in range(0, len(nums)):
            lookingFor = target - nums[i]
            if lookingFor in dict:
                return [dict[lookingFor], i]
            else:
                dict[nums[i]] = i
